index.html

```diff
@@ -0,0 +1,151 @@
+     <a href="#" class="nav-link">how it works</a>
+     <a href="#" class="nav-link">pricing</a>
+     </nav>
+
+
+   </div>
+
+   <form action=" https://www.freecodecamp.com/email-submit" id="form">
+    <p>
+      Handcrafted, home-made masterpieces
+    </p>
+    <input type="email" name="email" placeholder="Enter your email address" id="email">
+    <input type="submit" value="Get started" id="submit">
+  </form>
+<section id="features">
+
+  <div class="feature">
+    <img src="images\atom-solid.svg" alt="atom">
+    
+    <div>
+      <h3>
+        Premium Materials
+      </h3>
+      <p>
+        Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. 
+      </p>
+    </div>
+  </div>
+
+  <div class="feature">
+    <img src="images\battery-full-solid.svg" alt="battery">
+    
+    <div>
+      <h3>
+        Fast Shipping
+      </h3>
+      <p>
+        We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. 
+      </p>
+    </div>
+  </div>
+
+  <div class="feature">
+    <img src="images\truck-solid.svg" alt="truck">
+    
+    <div>
+      <h3>
+        Quality Assurance
+      </h3>
+      <p>
+        For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.
+      </p>
+    </div>
+  </div>
+</section>
+
+<section id="how-it-works">
+  <iframe width="560" id="video" height="315" src="https://www.youtube.com/embed/0OttfOXsP2M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+</section>
+
+<section id="pricing">
+<div class="price">
+<h4>
+  Tenor Trombone
+</h4>
+<span>
+  $600
+
+</span>
+<p>
+  
+  Lorem ipsum. <br>
+  Lorem ipsum. <br>
+  Lorem ipsum dolor. <br>
+  Lorem ipsum.
+
+</p>
+<a href="http://" class="select">select</a>
+</div>
+
+<div class="price">
+  <h4>
+    Bass Trombone
+  </h4>
+  <span>
+    $900
+  
+  </span>
+  <p>
+    
+    Lorem ipsum. <br>
+    Lorem ipsum. <br>
+    Lorem ipsum dolor. <br>
+    Lorem ipsum.
+  
+  </p>
+  <a href="http://" class="select">select</a>
+  </div>
+
+  <div class="price">
+    <h4>
+      Valve Trombone
+    </h4>
+    <span>
+      $1200
+    
+    </span>
+    <p>
+      
+      Lorem ipsum. <br>
+      Lorem ipsum. <br>
+      Lorem ipsum dolor. <br>
+      Lorem ipsum.
+    
+    </p>
+    <a href="http://" class="select">select</a>
+    </div>
+</section>
+
+<footer>
+  <a href="http://" target="_blank" rel="noopener noreferrer">privacy</a>
+
+  <a href="http://" target="_blank" rel="noopener noreferrer">terms</a>
+  <a href="http://" target="_blank" rel="noopener noreferrer">contact</a>
+</footer>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+
+    
+
+    
```

Comment:
- [ ] please endeavor to keep your code clean and professional. kindly remove the empty  space.